client/components/domains/registrant-extra-info/fr-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+	"type": "object",
+	"properties": {
+		"extra": {
+			"type": "object",
+			"properties": {
+				"sirenSiret": {
+					"type": "string",
+					"pattern": "^[0-9]*$",
+					"anyOf": [
+						{ "minLength": 9, "maxLength": 9 },
+						{ "minLength": 14, "maxLength": 14 }
+					]
+				},
+				"registrantType": {
+					"enum": [ "individual", "organization" ]
+				},
+				"registrantVatId": {
+					"type": "string",
+					"anyOf": [
+						{ "pattern": "^[A-Z]{2}.{2}[0-9]{4,8}[A-Z0-9]{2}" },
+						{ "pattern": "^.*GB.*$" }
+					]
+				}
+			}
+		}
+	},
```

Comment:
Love the sight of this.  This will be so much win when (heh) we have this for everything.